rundeckapp/grails-spa/packages/ui-trellis/src/components/execution-log/logViewer.vue

```diff
@@ -417,15 +417,11 @@ export default class LogViewer extends Vue {
         }, {passive: false})
     }
 
-    private handleExecutionLogResp(res: EnrichedExecutionOutput) {
-      this.execCompleted = res.execCompleted
-      this.completed = res.completed
-      this.nextProgress = res.completed ?
-        100:
-        Math.round((parseInt(res.offset) / res.totalSize) * 100)
-
-      if (res.entries.length == 0)
+    private handleExecutionLogResp() {
+      const defaultMaxLogSize = 3145728
+      if (this.maxLogSize == defaultMaxLogSize) {
```

Comment:
Seems like the default value is just used to compare with the value from controller to handle or not the execution log. It would be preferable to check if the property has been configured or not on rundeck-config.properties file

Reply:
@carlosrfranco code has been updated to use a separate property (trimOutput)